tsconfig-strict.json

```diff
@@ -720,6 +722,7 @@
     "extensions/interactions/NumericInput/directives/numeric-input-rules.service.spec.ts",
     "extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts",
     "extensions/interactions/NumericInput/directives/numeric-input-validation.service.spec.ts",
+    "extensions/interactions/PencilCodeEditor/directives/oppia-interactive-pencil-code-editor.component.ts",
```

Comment:
Can you add the spec file too?

Reply:
spec file doesn't exist for this component